README.md

```diff
@@ -7,18 +7,18 @@ can't be with them.
 
 ## How it works
 
-The emotion monitoring happens on a device—a laptop, or any IoT device
-that supports a camera—locally. Each frame from the camera is processed
+The emotion monitoring happens locally on a a laptop or any IoT device
```

Comment:
Can you fix the "a a"? Other than that, looks good.

Reply:
My bad; on it right now. Thanks for catching that.